subcommand/sync-catalog/command.go

```diff
@@ -9,9 +9,10 @@ import (
 	"os/signal"
 	"regexp"
 	"sync"
+	"syscall"
 	"time"
 
-	"github.com/deckarep/golang-set"
+	mapset "github.com/deckarep/golang-set"
```

Comment:
Can this change be unset please

Reply:
It's automatically added by `goimports`. I've removed it.